app-frontend/src/app/pages/projects/edit/colormode/colormode.controller.js

```diff
@@ -1,16 +1,78 @@
+/* global _ */
+
+const availableBands = [
+    {
+        label: 'Band 0',
+        value: 0
+    }, {
+        label: 'Band 1',
+        value: 1
+    }, {
+        label: 'Band 2',
+        value: 2
+    }, {
+        label: 'Band 3',
+        value: 3
+    }, {
+        label: 'Band 4',
+        value: 4
+    }, {
+        label: 'Band 5',
+        value: 5
+    }, {
+        label: 'Band 6',
+        value: 6
+    }, {
+        label: 'Band 7',
+        value: 7
+    }, {
+        label: 'Band 8',
+        value: 8
+    }, {
+        label: 'Band 9',
+        value: 9
+    }, {
+        label: 'Band 10',
+        value: 10
+    }, {
+        label: 'Band 11',
+        value: 11
+    }
+];
+
 export default class ProjectsEditColormode {
-    constructor($scope, colorCorrectService, projectService) {
+
+    constructor($scope, colorCorrectService, colorSchemeService, projectService) {
         'ngInject';
         this.$scope = $scope;
         this.$parent = $scope.$parent.$ctrl;
         this.colorCorrectService = colorCorrectService;
+        this.colorSchemeService = colorSchemeService;
         this.projectService = projectService;
     }
 
     $onInit() {
+        this.isLoading = true;
+        this.availableBands = availableBands;
         this.currentBands = null;
         this.correction = null;
-        this.$parent.sceneListQuery.then(() => {
+        this.defaultColorModes = {
+            custom: {
+                label: 'Custom',
+                value: {
+                    mode: 'custom-rgb',
+                    blueBand: 1,
```

Comment:
I set this up to match landsat natural color. The bands are actually in order of wavelength ascending. So blue is first. https://landsat.usgs.gov/what-are-band-designations-landsat-satellites